integration-tests/jest.config.js

```diff
@@ -1,12 +1,19 @@
+const glob = require(`glob`)
+
+const pkgs = glob
```

Comment:
Merely an alternate approach--both work well here!

```js
const roots = glob
  .sync(`*/`, {
    cwd: path.join(process.cwd(), `integration-tests`),
  })
  .map(testPath => `<rootDir>/integration-tests/${testPath}`)
```

Reply:
you probably want to replace `process.cwd()` with `__dirname`?